### PR TITLE
Queue fallback

### PR DESCRIPTION
**Pull request by zefciu**
If 'djid_reports' queue doesn't exist, fall back to 'default'

### CODE REVIEW

File: bob/djid/__init__.py [hunk 1]
```diff
@@ -29,12 +29,13 @@ class DjidMeta(type):
     """A djid metaclass."""
 
     __registry__ = {}
-    queue_name = 'djid_reports'
-    try:
-        queue = django_rq.get_queue('djid_reports')
-        connection = django_rq.get_connection(queue_name)
-    except KeyError:
-        queue = connection = None
+    for queue_name in ['djid_reports', 'default']:
```
Comment (mkurek) [inline bob/djid/__init__.py]: this whole for loop is a little bit overengineering for me. Wouldn't
```
queue = django_rq.get_queue('djid_reports' if 'djid_reports' in settings.RQ_QUEUES else 'default')
```
or 
```
try:
    queue = django_rq.get_queue(queue_name)
except KeyError:
    queue = django_rq.get_queue('default')
```
 be enough here?

Comment (ar4s) [inline bob/djid/__init__.py]: Can you move this block of code to the ``__init__``?

**Comment by mkurek** [comment]
:ok:

**Comment by kula1922** [comment]
:ok:

**Pull request merged**